pkg/antctl/raw/check/installation/command.go

```diff
@@ -0,0 +1,404 @@
+			Name: p.Name,
+			Labels: map[string]string{
+				"name": p.Name,
+				"kind": p.Role,
+			},
+		},
+		Spec: appsv1.DeploymentSpec{
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: p.Name,
+					Labels: map[string]string{
+						"name": p.Name,
+						"kind": p.Role,
+					},
+				},
+				Spec: corev1.PodSpec{
+					Containers: []corev1.Container{
+						{
+							Name: p.Name,
+							Env: []corev1.EnvVar{
+								{Name: "PORT", Value: fmt.Sprintf("%d", p.Port)},
+							},
+							Ports: []corev1.ContainerPort{
+								{ContainerPort: int32(p.Port)},
+							},
+							Image:           p.Image,
+							ImagePullPolicy: corev1.PullIfNotPresent,
+							Command:         p.Command,
+						},
+					},
+					Affinity:    p.Affinity,
+					Tolerations: p.Tolerations,
+				},
+			},
+			Replicas: &replicas32,
+			Selector: &metav1.LabelSelector{
+				MatchLabels: map[string]string{
+					"name": p.Name,
+					"kind": p.Role,
+				},
+			},
+		},
+	}
+}
+
+func NewTestContext(client kubernetes.Interface, config *rest.Config, clusterName string, o *options) *testContext {
+	return &testContext{
+		client:          client,
+		config:          config,
+		clusterName:     clusterName,
+		antreaNamespace: o.antreaNamespace,
+		namespace:       generateRandomNamespace(testNamespacePrefix),
+	}
+}
+
+func generateRandomNamespace(baseName string) string {
+	const letters = "abcdefghijklmnopqrstuvwxyz0123456789"
+	bytes := make([]byte, 5)
+	_, err := rand.Read(bytes)
+	if err != nil {
+		panic(err)
+	}
+	for i, b := range bytes {
+		bytes[i] = letters[b%byte(len(letters))]
+	}
+	return fmt.Sprintf("%s-%s", baseName, string(bytes))
+}
+
+func (t *testContext) teardown(ctx context.Context) {
+	t.Log("Deleting Post installation tests setup...")
+	t.client.CoreV1().Namespaces().Delete(ctx, t.namespace, metav1.DeleteOptions{})
+	t.Log("Waiting for Namespace %s to disappear", t.namespace)
+	err := wait.PollUntilContextTimeout(ctx, 1*time.Second, 5*time.Minute, true, func(ctx context.Context) (bool, error) {
+		_, err := t.client.CoreV1().Namespaces().Get(ctx, t.namespace, metav1.GetOptions{})
+		if err != nil {
+			return true, nil
+		}
+		return false, nil
+	})
+	if err != nil {
+		t.Log("Setup deletion failed")
+	} else {
+		t.Log("Setup deletion successful")
+	}
+}
+
+func (t *testContext) setup(ctx context.Context) error {
+	t.Log("Test starting....")
+	_, err := t.client.AppsV1().DaemonSets(t.antreaNamespace).Get(ctx, agentDaemonSetName, metav1.GetOptions{})
+	if err != nil {
+		return fmt.Errorf("unable to determine status of Antrea DaemonSet: %w", err)
+	}
+	t.Log("Creating Namespace for Post installation tests...")
+	_, err = t.client.CoreV1().Namespaces().Create(ctx, &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: t.namespace}}, metav1.CreateOptions{})
+	if err != nil {
+		return fmt.Errorf("unable to create Namespace %s: %s", t.namespace, err)
+	}
+	t.Log("Deploying echo-same-node service...")
```

Comment:
```suggestion
	t.Log("Deploying echo-same-node Service...")
```
In Antrea, we prefer to use the word with upper case for K8s and CRD objects., e.g.: Pod/Service/Deployment...
Please check other occurrences.